src/index.js

```diff
@@ -138,10 +138,15 @@ class Dropzone extends React.Component {
     this.isFileDialogActive = false;
 
     fileList.forEach((file) => {
-      if (this.fileAccepted(file) && this.fileMatchSize(file)) {
-        if (!disablePreview) {
+      if (!disablePreview) {
+        try {
           file.preview = window.URL.createObjectURL(file); // eslint-disable-line no-param-reassign
+        } catch (err) {
+          // Continue regardless of error
```

Comment:
I'm wondering if we should display a warning at least in a dev mode? I think you could re-use `invariant` here and wrap it into `process.env.NODE_ENV !== 'production'` condition. Thoughts?